website/src/components/Changelog/GUI.tsx

```diff
@@ -14,7 +14,14 @@ export default function GUI({ title }: { title: string }) {
   return (
     <Entries href={href} arches={arches} title={title}>
       {/* When you cut a release, remove any solved issues from the "known issues" lists over in `client-apps`. This must not be done when the issue's PR merges. */}
-      <Unreleased></Unreleased>
+      <Unreleased>
+        <ChangeItem enable={title === "Linux GUI"}>
+          This is a maintenance release with no user-facing changes.
+        </ChangeItem>
```

Comment:
Maybe we can just skip the changelog?

Reply:
It's there to fill space since the version number gets bumped on both OSes, right?